packages/auth/ios/RNFBAuth/RNFBAuthModule.m

```diff
@@ -1667,7 +1667,8 @@ - (NSDictionary *)firebaseUserToDict:(FIRUser *)user {
       @"uid" : hint.UID,
       @"factorId" : hint.factorID == nil ? [NSNull null] : [self getJSFactorId:(hint.factorID)],
       @"displayName" : hint.displayName == nil ? [NSNull null] : hint.displayName,
-      @"enrollmentDate" : enrollmentDate,
+      @"enrollmentTime" : enrollmentDate,
```

Comment:
Avoiding a breaking change here would be nice as changing the shape of the return isn't really an enabler of anything - so keeping backwards compat is worth it in my opinion

```suggestion
      @"enrollmentDate" : enrollmentDate, // from original implementation, kept to avoid breaking change
      @"enrollmentTime" : enrollmentDate,
```

Reply:
I've split this part of the change to https://github.com/invertase/react-native-firebase/pull/7653 in case you'd consider including it as a breaking change for v19 - also happy to stick with keeping runtime backwards compatibility with a deprecated property though.

The rest of this PR is additive, and the iOS part around adding `phoneNumber` definitely needs more testing / adjusting, so could come as a minor release later